docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -266,10 +160,92 @@ This is the reverse of the ``dict2items`` filter.
     {{ tags | items2dict(key_name='key', value_name='value') }}
 
 
+Discovering the data type
+-------------------------
+
+.. versionadded:: 2.3
+
+If you are unsure of the underlying Python type of a variable, you can use the ``type_debug`` filter to display it. This is useful in debugging when you need a particular type of variable::
+
+    {{ myvar | type_debug }}
+
+Setting the data type
+---------------------
+
+You can cast values as certain types. This is especially useful for ensuring that Ansible knows when to treat a value like "True" as a boolean and when to treat it as a string::
```

Comment:
```suggestion
You can cast values as certain types. This is especially useful for ensuring that Ansible knows when to treat a value like "True" as a Boolean and when to treat it as a string::
```

Reply:
TIL: Boolean is supposed to be capitalized... who knew?  :-)

Reply:
Huh. I bet we have a lot of places where that's wrong.